Source/IntegrationTests/TestFiles/LitTests/LitTest/traits/TraitResolution2.dfy.expect

```diff
@@ -6,19 +6,19 @@ TraitResolution2.dfy(101,14): Error: new can be applied only to class types (got
 TraitResolution2.dfy(103,14): Error: new can be applied only to class types (got C?<A?>)
 TraitResolution2.dfy(106,14): Error: new can be applied only to class types (got D?<A>)
 TraitResolution2.dfy(108,14): Error: new can be applied only to class types (got D?<A?>)
-TraitResolution2.dfy(110,14): Error: new can be applied only to class types (got D<A>)
-TraitResolution2.dfy(112,14): Error: new can be applied only to class types (got D<A?>)
+TraitResolution2.dfy(110,14): Error: new can be applied only to class types (got D?<A>.Init<A>)
```

Comment:
I think these error messages made more sense before this change - what do you think? And would it be easy to restore the previous format?

Reply:
> I think these error messages made more sense before this change - what do you think?

Well, they looked prettier but they were confusing because before it said `Error: new can be applied only to class types (got D<A>)`, while `D<A>` is a valid target for `new`. The new error at least reveals the problem: that `D?` is used as a target for `new`.

Reply:
Reverted back to the old behavior, with a fix for the missing `?`